Program.cs

```diff
@@ -1,57 +1,396 @@
+    /// Date: February 21, 2021
+    /// </summary>
     class Program
     {
+
+        /// <summary>
+        /// The main.
+        /// </summary>
+        /// <param name="args">The args.</param>
         static void Main(string[] args)
         {
             Console.WriteLine("Canadian Cities Data Analayzer © Hunter Bennett, Connor Black, James Dunton");
-            char selection;
+
+            Statistics citiesStats = StartMenu();
+            bool valid = false;
+            citiesStats.DisplayProvinceList();
+            do
+            {
+                string selection;
+                string titleText = "Mode Selection Menu";
+                string dash = new string('-', titleText.Length);
+                Console.WriteLine($"{dash}\n{titleText}\n{dash}\n");
+                Console.WriteLine("\nAvailable selections:");
+                Console.WriteLine("\t1) Display city statistics menu.");
+                Console.WriteLine("\t2) Display province statistics menu.");
+                Console.Write("Please make a selection: ");
+                selection = Console.ReadLine();
+                switch (selection.ToLower())
+                {
+                    case "1":
+                        CityMenu(citiesStats);
+                        break;
+                    case "2":
+                        ProvinceMenu(citiesStats);
+                        break;
+                    case "exit":
+                        System.Environment.Exit(0);
+                        break;
+                    case "restart":
+                        citiesStats = StartMenu();
+                        break;
+                    default:
+                        Console.Write("\nInvalid selection, please key in a valid selection: ");
+                        break;
+                }
+            } while (!valid);
+
+            citiesStats.DisplayLargestPopulationCity("Ontario");
+            citiesStats.DisplaySmallestPopulationCity("Ontario");
+            citiesStats.DisplayProvincePopulation("Ontario");
+            citiesStats.DisplayCityInfo("London");
+            citiesStats.DisplayCityInfo("Windsor");
+            citiesStats.DisplayProvinceCities("Ontario");
+
+            //TODO: implement options
+        }
+        /// <summary>
+        /// First menu that the user sees.
+        /// </summary>
+        /// <returns></returns>
+        static Statistics StartMenu()
+        {
+            Statistics citiesStats = null;
+            bool valid = false;
+            string titleText = "Main Menu";
+            string selection;
+            string dash = new string('-', titleText.Length);
             do
             {
+                Console.WriteLine($"\n{dash}\n{titleText}\n{dash}\n");
+                Console.WriteLine("-Key in 'exit' to exit the application at any menu in this program.");
+                Console.WriteLine("-key in 'restart' at any menu to return to this menu.");
```

Comment:
Not a huge issue, but you forgot to capitalize the k. Just more of a consistency thing